arangod/Cluster/ClusterFeature.h

```diff
@@ -98,11 +98,11 @@ class ClusterFeature : public application_features::ApplicationFeature {
   /// - "jwt-write"  = JWT required to access post/put/delete operations
   /// - "jwt-compat" = compatibility mode = same permissions as in 3.7
   std::string const& apiJwtPolicy() const noexcept { return _apiJwtPolicy; }
-  
-  Counter& followersDroppedCounter() { return _followersDroppedCounter->get(); }
-  Counter& followersRefusedCounter() { return _followersRefusedCounter->get(); }
-  Counter& followersWrongChecksumCounter() { return _followersWrongChecksumCounter->get(); }
-  Counter& followersTotalRebuildCounter() { return _followersTotalRebuildCounter->get(); }
+
+  metric::Counter& followersDroppedCounter() { return _followersDroppedCounter->get(); }
+  metric::Counter& followersRefusedCounter() { return _followersRefusedCounter->get(); }
+  metric::Counter& followersWrongChecksumCounter() { return _followersWrongChecksumCounter->get(); }
+  metric::Counter& followersTotalRebuildCounter() { return _followersTotalRebuildCounter->get(); }
```

Comment:
The PR description of this PR states:
> remove std::optional<std::reference_wrapper<SomeMetricType>>, which is no different from T* since the operator-> that is used in our code is noexcept

Is it intentional that options with refererence_wrappers are still used here, or should they be changed here as well?

There are also additional places in other files where reference wrappers are still used. I will not mention them separately.

Reply:
> But I want:

What is written in these paragraphs isn't what I did in this PR.
This is what I will want to do. This is some motivation for this PR.

Reply:
I don’t do it in this PR because it seems like it might cause discussion.
But this PR is big and difficult to maintain and review.
So I decided not to make it bigger.

Reply:
I have updated the description to make it clearer.

Reply:
Thanks, got it.